apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -114,7 +115,7 @@ class GqlWithKnexLoadList {
  * @param {String[]} sortBy
  * @param {Number} chunkSize
  * @param {Number} limit
- * @param {function(Array): Array} chunkProcessor A place to use or/and modify just loaded chunk
+ * @param {function(Array): Array | Promise<Array>} chunkProcessor A place to use or/and modify just loaded chunk
  * @returns {Promise<*[]>}
  */
 const loadListByChunks = async ({
```

Comment:
I don't like an idea to use this function for processing data
The name of the function is starting with **load**
It's like to add another argument and it will start saving models by chunks


Maybe, better is to make a separated function for this

Reply:
The `chunkProcessor` allows modifying loaded data on the fly. One of the options is to filter chunks or return an empty array.
Load is not equal to return. Load from the database and optionally pass through chunkProcessor.

The separated function will have absolutely the same functionality. Btw, current functionality is used in many places. I just have to add the ability to pass the async function.